servers/cu/src/domain/client/ao-process.test.js

```diff
@@ -1287,5 +1288,42 @@ describe('ao-process', () => {
     })
   })
 
+  describe('createProcessMemoryCache', () => {
+    const THIRTY_TWO_DAYS = 1000 * 60 * 60 * 24 * 30
+    test('can set cache TTL greater than int32 max value without overflowing', async () => {
+      const lt = (await import('long-timeout')).default
+      let setTimeoutCalled = false
+      let ttlRan = false
+      let timeoutRef
+      const cache = await createProcessMemoryCache({
+        TTL: THIRTY_TWO_DAYS,
+        MAX_SIZE: bytes('500mb'),
+        setTimeout: (cb, time) => {
+          console.log('setting timeout')
+          setTimeoutCalled = true
+          timeoutRef = lt.setTimeout(() => {
+            ttlRan = true
+            cb()
+          }, time)
+          return timeoutRef
+        },
+        gauge: () => false
+      })
+      // call something on the cache
+      cache.set('foo', {
+        File: new File([], 'hello world'),
+        Memory: Buffer.from('hello world')
+      })
+      // wait to see that this implementation does not overflow
+      await new Promise((resolve) => setTimeout(resolve, 50))
+
+      assert.ok(setTimeoutCalled)
+      assert.ok(!ttlRan)
+
+      timeoutRef.unref()
+      lt.clearTimeout(timeoutRef)
```

Comment:
If you inject `clearTimeout`, then I think you can call just `lt.clearTimeout` here.